app/src/main/java/org/jellyfin/androidtv/integration/LeanbackChannelWorker.kt

```diff
@@ -154,14 +153,14 @@ class LeanbackChannelWorker(
 			userId = user.id
 			imageTypeLimit = 1
 			limit = 10
-			fields = arrayOf(ItemFields.DateCreated)
 		})
 
 		// Add new items
-		response?.items?.let { items ->
-			context.contentResolver.bulkInsert(
-				WatchNextPrograms.CONTENT_URI,
-				items.map { item -> getBaseItemAsWatchNextProgram(item).toContentValues() }.toTypedArray()
+		response?.items?.reverse()
```

Comment:
We should specify the sorting in the `getNextUpEpisodes()` call. The sorting is already correct for me though.

Reply:
For me the ten episodes that load into the Play Next channel are the same as the first ten episodes in my Next Up row, but instead of sorting by most recently watched like inside the app, the channel sorts by the episode’s Date Added value. e.g. my first entry in the Next Up row inside the app is the eighth entry in the Play Next channel

Reply:
I just checked this and you're correct. I think this can be fixed by changing the sort order in that case.

Reply:
It looks like the Play Next channel order is being determined by `setLastEngagementTimeUtcMillis(item.dateCreated?.time ?: System.currentTimeMillis())`, which is why removing ItemFields.DateCreated from the NextUpQuery caused the order to change (it made setLastEngagement fall back to System.currentTime). 

I tried `setLastEngagementTimeUtcMillis(apiClient.getItem(item.seriesId)?.userData?.lastPlayedDate?.time ?: System.currentTimeMillis())` but it just falls back to System.currentTime. Do you have any ideas?